docs-src/go/auto-restart-worker-tip.md

```diff
@@ -0,0 +1,20 @@
+---
+id: auto-restart-worker-tip
+title: Automatically restart Worker Process during development
+description: Developer tip, is to use the gow package to automatically restart the Worker Process.
```

Comment:
Just "use the gow package to automatically restart the Worker Process"? Or "Developer tip: Use the gow package to automatically restart the Worker Process."